docs/en_US/gsoc-ideas.md

```diff
@@ -0,0 +1,64 @@
+
+Enjoy your summer of code!
+
+## 1. CLI tool
+
+* Project Description: The primary method to manage CRDs of ob-operator is manipulating YAML manifests which is not easy enough for new-coming users to get started. Within the CLI tool, features like component installation, demo setup,  resource management and necessary validation can be implemented.
+* Required Skills: Golang, Kubernetes
+* Project Size: medium/large
+* Expected Outcomes: A complete CLI tool to control and manage CRDs in ob-operator, in other words, to manage clusters, tenants, backups on ob-operator in kubernetes cluster. Or a module in the CLI tool.
+* References: 
+  * [kubernetes/client-go](https://github.com/kubernetes/client-go)
+  * [spf13/cobra](https://github.com/spf13/cobra)
+  * [manifoldco/promptui](https://github.com/manifoldco/promptui)
+
+
+## 2. Light-weighted Operations
+
+* Project Description: Use Light-weighted operation task types to influence the status of resources instead of modifying specifications of those bigger resources.
+* Required Skills: Golang, Development of Kubernetes controller, Docker
+* Project Size: medium
+* Expected Outcomes: Implementation of CRD(s) and corresponding controller(s) for cluster and tenant, which can trigger small operational actions and reveal the progress through `status` fields.
+* References: 
+  * [Tenant operation: Failover](https://en.oceanbase.com/docs/common-oceanbase-database-10000000001106036)
+  * [Tenant operation: Replay log](https://en.oceanbase.com/docs/common-oceanbase-database-10000000001103949)
+  * [Architecture of ob-operator](https://oceanbase.github.io/ob-operator/docs/en_US/arch.html)
+
+
+## 3. Alertmanager Integration
+
+* Project Description: There are some features about cluster management in OceanBase dashboard, a web-based dashboard application, and tenant management features are coming soon. OceanBase dashboard has bundled prometheus to manage time series metrics data by now. It's recommended that Alertmanager performs alerting tasks along with promethues. Integrating Alertmanager into OceanBase dashboard could be an attracive feature.
+* Required Skills: Web development(React), Kubernetes, Prometheus, Alertmanager
+* Project Size: medium
+* Expected Outcomes: A functional panel in OceanBase dashboard in which users could view alert events, configure alert rules, and define alert templates in the scope of Alertmanager.
+* References: 
+  * [Alertmanager](https://prometheus.io/docs/alerting/latest/alertmanager/)
+  * [prometheus/alertmanager](https://github.com/prometheus/alertmanager)
+  * [OceanBase - Monitor - Overview](https://en.oceanbase.com/docs/common-oceanbase-database-10000000001103563)
+
+
+## 4. OceanBase Database Proxy integration
```

Comment:
it's not integration, it's a feature in oceanbase dashboard
how about "ODP management in OceanBase Dashboard"